astroplan/core.py

```diff
@@ -26,10 +26,11 @@
            "Constraint", "TimeWindow", "AltitudeRange",
            "AboveAirmass", "MAGIC_TIME"]
 
-#__doctest_requires__ = {'*': ['scipy.integrate']}
```

Comment:
Can you please leave a `TODO` comment here to make it easier to update this in the future?
```
# TODO: remove this statement once the moon is implemented without pyephem
```